includes/class-wc-google-gtag-js.php

```diff
@@ -26,7 +26,7 @@ class WC_Google_Gtag_JS extends WC_Abstract_Google_Analytics_JS {
 	private $mappings = array(
 		'begin_checkout'   => 'woocommerce_before_checkout_form',
 		'purchase'         => 'woocommerce_thankyou',
-		'view_item_list'   => 'woocommerce_before_shop_loop_item',
+		'view_item_list'   => 'woocommerce_loop_add_to_cart_link',
```

Comment:
Changing this breaks the shop pages, because we switch it from an action to a filter. While actions and filters are interchangeable in most cases the important difference is that a filter must return a value otherwise the next filter will get a blank value. This causes the add to cart buttons to disappear.

We need to either pick a filter here. Or handle this separately so we can use `add_filter` in the function `map_actions`.